tests/functional/aws-node-sdk/test/multipleBackend/objectTagging.js

```diff
@@ -154,6 +154,9 @@ function mpuWaterfall(params, cb) {
 
 describeSkipIfNotMultiple('Object tagging with multiple backends',
 function testSuite() {
+    if (!process.env.S3_END_TO_END) {
```

Comment:
why do we need this condition?

Reply:
I had followed the form of what has been added to master, such that retries should not be allowed when the var is set. I see that it won't be used in rel/6.3, I will make the proper changes.